requirements-dev.txt

```diff
@@ -1,6 +1,6 @@
 # Keep the below block the same as requirements.txt
 depthai==2.26.0
-viam-sdk==0.25.2
+viam-sdk==0.29.3
```

Comment:
https://github.com/viamrobotics/viam-python-sdk/releases/tag/v0.29.3

fixes a shutdown issue

Reply:
honestly not sure why it's showing as a diff though

we are on 0.29.3 on main

Reply:
make sure you've rebased onto main in case.